README.rst

```diff
@@ -307,11 +296,15 @@ Since version 2.0, the following functions are also available in the ``xdist`` m
         :param request_or_session: the `pytest` `request` or `session` object
         """
 
+    def is_xdist_master(request_or_session) -> bool:
+        """Deprecated alias for is_xdist_controller."""
+
     def get_xdist_worker_id(request_or_session) -> str:
         """Return the id of the current worker ('gw0', 'gw1', etc) or 'master'
         if running on the controller node.
 
-        If not distributing tests (for example passing `-n0` or not passing `-n` at all) also return 'master'.
+        If not distributing tests (for example passing `-n0` or not passing `-n` at all)
+        also return 'master'.

```

Comment:
A bit confusing that `get_xdist_worker_id` returns `"master"` instead of `"controller"`, but I'm sure it is not worth changing it at this point.

Reply:
I figure it was kept as "master" for backward compat.